config/robots.rb

```diff
@@ -2024,4 +2024,103 @@
                    target_state: 'passed'
                  }
                })
+
+  # Robots for pWGS 384 pipeline
+
+  custom_robot('star-pwgs-384-post-shear-to-pwgs-384-post-shear-xp',
+               name: 'STAR pWGS-384 Post Shear => pWGS-384 Post Shear XP',
+               beds: {
+                 bed(12).barcode => {
+                   purpose: 'LB Post Shear',
+                   states: ['passed'],
+                   label: 'Bed 12'
+                 },
+                 bed(13).barcode => {
+                   purpose: 'LB Post Shear',
+                   states: ['passed'],
+                   label: 'Bed 13'
+                 },
+                 bed(14).barcode => {
+                   purpose: 'LB Post Shear',
+                   states: ['passed'],
+                   label: 'Bed 14'
+                 },
+                 bed(15).barcode => {
+                   purpose: 'LB Post Shear',
+                   states: ['passed'],
+                   label: 'Bed 15'
+                 },
+                 bed(7).barcode => {
+                   purpose: 'pWGS-384 Post Shear XP',
+                   states: ['pending'],
+                   label: 'Bed 7',
+                   parents: [bed(12).barcode, bed(13).barcode, bed(14).barcode, bed(15).barcode],
+                   target_state: 'passed'
```

Comment:
I see robot: hamilton-star-pf-post-shear-to-pf-post-shear-xp-384
with an attribute class: 
```ruby
class: 'Robots::QuadrantRobot'
```
by naming, could it be related with what they want to do (transfer in quadrants of the 4 source plates)?
This is in the docs of that class:
```ruby
  # Quadrant robots take up to four parent plates which
  # are numbered according to their quadrant.
  # eg. A1 => 1, B1 => 2, A1 => 3, B2 => 4
  # While the pooling robot will work for the majority of cases,
  # it exhibits incorrect behaviour if Well A1 is missing on any
  # of the source wells.
```

Reply:
I will add this class, and also the `destination_bed` field